presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -378,53 +385,55 @@ void PrestoServer::run() {
   // down.
   httpServer_->start(getHttpServerFilters());
 
-  LOG(INFO) << "SHUTDOWN: Stopping all periodic tasks...";
+  PRESTO_SHUTDOWN_LOG(INFO) << "Stopping all periodic tasks...";
   periodicTaskManager_->stop();
 
   // Destroy entities here to ensure we won't get any messages after Server
   // object is gone and to have nice log in case shutdown gets stuck.
-  LOG(INFO) << "SHUTDOWN: Destroying Task Resource...";
+  PRESTO_SHUTDOWN_LOG(INFO) << "Destroying Task Resource...";
   taskResource_.reset();
-  LOG(INFO) << "SHUTDOWN: Destroying Task Manager...";
+  PRESTO_SHUTDOWN_LOG(INFO) << "Destroying Task Manager...";
   taskManager_.reset();
-  LOG(INFO) << "SHUTDOWN: Destroying HTTP Server...";
+  PRESTO_SHUTDOWN_LOG(INFO) << "Destroying HTTP Server...";
   httpServer_.reset();
 
+  unregisterConnectors();
```

Comment:
@xiaoxmeng 
What is inside connectors that causes crashes?
Is it they try accessing memory manager/allocation after it is destroyed or something else?